lib/entities/V2HardQuoteResponse.ts

```diff
@@ -0,0 +1,42 @@
+import { BigNumber } from "ethers";
+import { HardQuoteResponse } from "./HardQuoteResponse";
+import { CosignedV2DutchOrder } from "@uniswap/uniswapx-sdk";
+
+export class V2HardQuoteResponse extends HardQuoteResponse<CosignedV2DutchOrder> {
+    public toLog() {
+        return {
+          quoteId: this.quoteId,
+          requestId: this.requestId,
+          tokenInChainId: this.chainId,
+          tokenOutChainId: this.chainId,
+          tokenIn: this.tokenIn,
+          amountIn: this.amountIn.toString(),
+          tokenOut: this.tokenOut,
+          amountOut: this.amountOut.toString(),
+          swapper: this.swapper,
+          filler: this.filler,
+          orderHash: this.order.hash(),
+          createdAt: this.createdAt,
+          createdAtMs: this.createdAtMs,
+        };
+    }
+
+    public get amountOut(): BigNumber {
```

Comment:
This returns the current amount on the auction curve (via `getDecayedAmount()` in the sdk). Why don't we do something similar for v3?

Reply:
The only place this is used is in logging, and for V3 I just directly logged the input, which gives starting amount and curve